src/test/java/me/itzg/helpers/modrinth/TestProjectRef.java

```diff
@@ -0,0 +1,90 @@
+package me.itzg.helpers.modrinth;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import me.itzg.helpers.modrinth.model.VersionType;
+
+public class TestProjectRef {
+    private String expectedSlug = "test_project1";
+    private String expectedVersionName = "test_version1";
+    private ProjectRef projectRefUT;
+
+    @Test
+    void testProjectRefHasProjectSlug() {
+        projectRefUT = new ProjectRef(this.expectedSlug, null);
+
+        assertEquals(expectedSlug, projectRefUT.getIdOrSlug());
```

Comment:
Please switch to https://assertj.github.io/doc/ rather than using the junit assertions. It'll be consistent with the rest of the project and they are much more expressive.